python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -730,10 +767,15 @@ def _init(self, *args, **kwds):
     """)
 
     def _str(self, interactive=False):
+        #import time
+        #b4=time.time()
         import cStringIO
         sio = cStringIO.StringIO()
         stripProxy(self).printSummaryTree(0, 0, '', out=sio, interactive=interactive)
-        return str(sio.getvalue()).rstrip()
+        returnable = str(sio.getvalue()).rstrip()
+        #a4=time.time()
+        #logger.info("Time Taken to generate printSummaryTree: %s" % str(a4-b4))
+        return returnable
```

Comment:
Fair enough, I've commented this back in, in a private commit anyway as I was seeing bottlenecks in the summary tree function.
Any feeling if this is wanted or not?